File: PRAKTIKUM-PEMROGRAMAN1/MODUL6/Python/PRAK603-2410817210024-MuhammadIbnuSina.py
n1, n2 = map(int, input().split())

if n1!=n2:
    print("Jumlah tidak sama")
else:
    matriksa = list(map(int, input().split()))
    matriksb = list(map(int, input().split()))

    for i in range (n1):
        print(matriksa[i] * matriksb[i], end=" ")

File: PRAKTIKUM-PEMROGRAMAN1/MODUL6/C/PRAK603-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
    } else {
        int matriksa[n1], matriksb[n2];

        for (int i = 0; i < n1; i++) {
            scanf("%d", &matriksa[i]); }

        for (int i = 0; i < n2; i++) {
            scanf("%d", &matriksb[i]); }

        for (int i = 0; i < n1; i++) {
            printf("%d ", matriksa[i] * matriksb[i]); } 
    }
    return 0;
}